esp/src/eclwatch/VizWidget.js

```diff
@@ -391,7 +391,7 @@ define([
 
         refreshData: function () {
             if (this.limit.get("value") > this.rows.length) {
-                var result = this.wu.results[this.params.Sequence];
+                var result = this.wu.results.filter(r => r.Sequence == this.params.Sequence)[0];
```

Comment:
This was definitely an array out of bounds problem. I think there might be some other areas in this widget where it crashes gracelessly, but I couldn't quite reproduce it reliably (it could just have something to do with if the result(s) selected can actually be visualized..?).